apps/gateway/.env.example

```
@@ -1,11 +1,4 @@
 PORT=3002
-
-# Stack Auth keys
-# Get these variables by creating a project on https://app.stack-auth.com.
-STACK_SECRET_SERVER_KEY=your-stack_secret_server_key
-VITE_STACK_PROJECT_ID=your-stack_project_id
-
-# Model API Keys
-GROQ_API_KEY=your-groq_api_key
-VOYAGE_API_KEY=your-voyage_api_key
\ No newline at end of file
+VERTEX_LOCATION=us-central1
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)